src/ccl_core.c

```diff
@@ -511,8 +512,8 @@ void ccl_cosmology_set_status_message(ccl_cosmology * cosmo, const char * messag
   #pragma omp critical
   {
     if(strlen(cosmo->status_message) != 0) {
-      ccl_raise_warning(CCL_ERROR_OVERWRITE, "Status message being overwritten.");
-      ccl_raise_warning(cosmo->status, cosmo->status_message);
+      ccl_raise_warning(CCL_ERROR_OVERWRITE, "Status message being overwritten:");
+      fprintf(stderr, "STATUS: %d. %s\n", cosmo->status, cosmo->status_message);
```

Comment:
Maybe you already discussed this. Why are we changing this from a warning to always printing an error message?

Reply:
Right, but it's enclosed by:
```
if (_ccl_debug_mode_policy == CCL_DEBUG_MODE_ON) {
    fprintf(stderr, "WARNING %d: %s\n", err, message);
  }
```
meaning that we can switch them off, right? Don't we want the same here?

Reply:
I'd say no, since this "warning" only gets printed when there's an error message being overwritten. So in that sense it should be an error but our current error handling can't deal with more than one error occurring.